src/d3a/models/market/grid_fees/constant_grid_fees.py

```diff
@@ -0,0 +1,89 @@
+from d3a.models.market.grid_fees import BaseClassGridFees
+from d3a.models.market.market_structures import TradeBidInfo
+
+
+class ConstantGridFees(BaseClassGridFees):
+
+    @staticmethod
+    def update_incoming_bid_with_fee(source_bid, original_bid, tax_ratio=None):
```

Comment:
Better rename `tax_ratio` to something like `tax_value` as it is targeting the constant grid fee here

Reply:
Tax_ratio is not used here. This function is mainly used in the percentage based grid fees. Both Classes have to implement this function because of the basclass.

Reply:
But I agree with renaming tax_ratio throughout the class. It is indeed a grid_fee_rate, right?

Reply:
You are correct, this is a grid_fee_rate, and it should be renamed throughout the class hierarchy (in ABC, base_model and here). Will rename this, thanks